.circleci/config.yml

```diff
@@ -84,13 +84,16 @@ jobs:
           export ST2_HOSTNAME=$(ifconfig docker0 | grep 'inet addr' | awk -F: '{print $2}' | awk '{print $1}')
           docker-compose run \
           -e ST2_HOSTNAME=${ST2_HOSTNAME} \
-          -e ST2_USERNAME=${ST2_USERNAME} \
+          -e ST2_USERNAME=${ST2_USER} \
           -e ST2_PASSWORD=${ST2_PASSWORD} \
           -e SLACK_TOKEN=${SLACK_TOKEN} \
           ${DISTRO}-test test
     - run:
-        name: Collect logs
-        command: for name in $(docker ps -a --format "{{.Names}}"); do docker logs ${name} > /packages/${DISTRO}/log/${name}.log 2>&1; done
+        name: Collect logs and artifacts
+        command: |
+          for name in $(docker ps -a --format "{{.Names}}"); do docker logs ${name} > /tmp/st2chatops/${DISTRO}/logs/${name}.log 2>&1; done
+          mkdir -p ~/packages/${DISTRO}
+          cp -r /tmp/st2chatops/${DISTRO}/logs  ~/packages/${DISTRO}
```

Comment:
This part is still not properly synced up with the https://github.com/StackStorm/st2chatops/pull/115
Take a look again.

Reply:
The original code copies things to `/packages/${DISTRO}` (relative to the filesystem ROOT), and the new code correctly copies things to `~/packages/${DISTRO}` (relative to the user's home directory).

This also copies the log files to the `~/packages/${DISTRO}/logs` directory.

Can you verify that this is correct behavior?

Reply:
Compare with the changes made in https://github.com/StackStorm/st2chatops/pull/115/files

here we don't need to `cp -r /tmp/st2chatops/${DISTRO}/logs  ~/packages/${DISTRO}`, but instead write logs to required dir: 
https://github.com/StackStorm/st2chatops/blob/9f7b79acaafb60df5c627096ea640badd5eb9344/.circleci/config.yml#L93

Reply:
Gotcha, thank you for explaining - working on it.

Reply:
Resolved.